src/components/MyCards/MyCards.test.js

```diff
@@ -33,50 +33,50 @@ describe('on clicking an enabled card', () => {
         const playCard = jest.spyOn(MyCards.prototype, 'playCard');
         const wrapper = mount(
             <Provider store={store}>
-                <MyCards fetchCards={jest.fn()} myCards={cards} />
+                <MyCards fetchCards={jest.fn()} cards={cards} />
             </Provider>
         );
         wrapper.find({ 'data-cy': 'card' }).first().simulate('click');
         expect(playCard).toHaveBeenCalled();
         playCard.mockRestore();
     });
 
-    describe('on playWordAndCard flag true', () => {
-        it('calls requestPlayCard', (() => {
-            jest.spyOn(MyCards.prototype, 'isEnabled').mockImplementation(() => true);
-            const requestPlayCard = jest.fn();
-            const wrapper = mount(
-                <Provider store={store}>
-                    <MyCards fetchCards={jest.fn()} myCards={cards} requestPlayCard={requestPlayCard} playWordAndCard={true} />
-                </Provider>
-            );
-            wrapper.find({ 'data-cy': 'card' }).first().simulate('click');
-            expect(requestPlayCard).toHaveBeenCalled();
-            requestPlayCard.mockRestore();
-        }));
-    });
+    // describe('on playWordAndCard flag true', () => {
```

Comment:
Are we gonna use this test eventually? Why is it commented out?

Reply:
This flag hasn't been used for a while and so I removed it in the refactor, the test is redundant now